packages/fxa-auth-server/lib/senders/emails/templates/passwordResetAccountRecovery/index.mjml

```diff
@@ -0,0 +1,42 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-include path="./css/global.css" type="css" css-inline="inline" />
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="passwordResetAccountRecovery-title">Your account password was reset with a recovery key</span>
+    </mj-text>
+
+    <mj-text css-class="text-body">
+      <span data-l10n-id="passwordResetAccountRecovery-description">You have successfully reset your password using a recovery key from the following device:</span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<%- include('/partials/location/index.mjml') %>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-body">
+      <span data-l10n-id="passwordResetAccountRecovery-regen-required">You will need to generate a new recovery key.</span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<%- include('/partials/button/index.mjml') %>
+
+<mj-section>
```

Comment:
I think we can just include the `automatedEmailResetPassword` MJML partial here? Same question for `postAddAccountRecovery` MJML file.